### PR TITLE
Add more examples for BOOL aggregations to docs

### PR DESCRIPTION
**Pull request by igormunkin**
The reference for `BOOL_AND`/`BOOL_OR`/`BOOL_XOR` aggregations missing several examples for `NULL`-related behaviour. Furthermore, `NULL`-related peculiarities was not even described in the english alternative (as well as the signatures).

### Changelog category

* Documentation (changelog entry is not required)

### CODE REVIEW

File: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
+```yql
+BOOL_AND(Bool?)->Bool?
+BOOL_OR(Bool?)->Bool?
+BOOL_XOR(Bool?)->Bool?
+```
+
 Apply the relevant logical operation  (`AND`/`OR`/`XOR`) to all values in a Boolean column or expression.
 
-These functions **don't skip** `NULL` during aggregation, with even one `NULL` turning the result into `NULL`. To skip `NULLs` during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.
+These functions **don't skip** `NULL` during aggregation and follows the next rule: `true and null == null`, `false or null == null`. For `BOOL_AND` with any `true` values and even one `NULL` the result is `NULL`, but even one `false` changes it to `false` regardless whether `NULL` is present. For `BOOL_OR` with any `false` valse and even one `NULL` the result is `NULL`, but even one `true` changes it to `true` regardless whether `NULL` is present. `BOOL_XOR` results to `NULL` if any `NULL` is found. One can find more examples of such behaviour below.
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md:13]:
```suggestion
Unlike most other aggregate functions, these functions **don't skip** `NULL` during aggregation and use the following rules: 

- `true AND null == null`
- `false OR null == null`

For `BOOL_AND`:

- If there is any `true` value and even one `NULL`, the result is `NULL`.
- If there is even one `false`, the result changes to `false`, regardless of whether `NULL` is present.

For `BOOL_OR`:

- If there is any `false` value and even one `NULL`, the result is `NULL`.
- If there is even one `true`, the result changes to `true`, regardless of whether `NULL` is present.

For `BOOL_XOR`:

- The result is `NULL` if any `NULL` is found.

Examples of such behavior can be found below.
```

File: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md [hunk 2]
```diff
@@ -1,17 +1,42 @@
...
+To skip `NULLs` during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.
 
 ### Examples
 
 ```yql
+$data = AsList(
+    AsStruct(true  AS nonNull, true AS nonFalse, NULL  AS nonTrue, true  AS anyVal),
+    AsStruct(false AS nonNull, NULL AS nonFalse, NULL  AS nonTrue, NULL  AS anyVal),
+    AsStruct(false AS nonNull, NULL AS nonFalse, false AS nonTrue, false AS anyVal),
+);
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md:24]:
```suggestion
$data = [
    <|nonNull: true, nonFalse: true, nonTrue: NULL, anyVal: true|>,
    <|nonNull: false, nonFalse: NULL, nonTrue: NULL, anyVal: NULL|>,
    <|nonNull: false, nonFalse: NULL, nonTrue: false, anyVal: false|>,
];
```

File: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md [hunk 3]
```diff
@@ -1,17 +1,42 @@
...
+BOOL_OR(Bool?)->Bool?
+BOOL_XOR(Bool?)->Bool?
+```
+
 Apply the relevant logical operation  (`AND`/`OR`/`XOR`) to all values in a Boolean column or expression.
 
-These functions **don't skip** `NULL` during aggregation, with even one `NULL` turning the result into `NULL`. To skip `NULLs` during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.
+These functions **don't skip** `NULL` during aggregation and follows the next rule: `true and null == null`, `false or null == null`. For `BOOL_AND` with any `true` values and even one `NULL` the result is `NULL`, but even one `false` changes it to `false` regardless whether `NULL` is present. For `BOOL_OR` with any `false` valse and even one `NULL` the result is `NULL`, but even one `true` changes it to `true` regardless whether `NULL` is present. `BOOL_XOR` results to `NULL` if any `NULL` is found. One can find more examples of such behaviour below.
+
+To skip `NULLs` during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.
```
Comment (blinkov) [inline ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md:15]:
```suggestion
To skip `NULL` values during aggregation, you can use the functions `MIN`/`MAX` or `BIT_AND`/`BIT_OR`/`BIT_XOR`.
```

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md (hunk 2 above)
Comment (igormunkin) [reply ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md]: [Fixed](https://github.com/ydb-platform/ydb/pull/12566/commits/506b93bb6dfefd036f001e6eec102d4bf67dc62d).

File: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md (hunk 3 above)
Comment (igormunkin) [reply ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md]: [Fixed](https://github.com/ydb-platform/ydb/pull/12566/commits/200750bf8893df78e95364c1a64fe9902d0827c2).

File: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md (hunk 1 above)
Comment (igormunkin) [reply ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/aggregation/bool_bit.md]: [Fixed](https://github.com/ydb-platform/ydb/pull/12566/commits/5291e5a09c3c91d3c9ec9c98b8cbb9b0dd3f51a3) along with /ru version.

**Review by fomichev3000 — APPROVED** [review]

**Review by anton-bobkov — APPROVED** [review]

**Pull request merged**